build.sbt

```diff
@@ -29,7 +29,13 @@ lazy val test = Seq(
   "org.pegdown" % "pegdown" % "1.6.0" % scope,
   "org.scalatest" %% "scalatest" % "3.0.4" % scope,
   "uk.gov.hmrc" %% "hmrctest" % "3.1.0" % scope,
-  "uk.gov.hmrc" %% "reactivemongo-test" % "3.0.0" % scope
+  "uk.gov.hmrc" %% "reactivemongo-test" % "3.0.0" % scope,
+  "org.scalatest" %% "scalatest" % "3.0.4" % scope,
+  "org.scalatestplus.play" %% "scalatestplus-play" % "2.0.1" % scope,
+  "org.mockito" % "mockito-core" % "2.11.0" % scope,
```

Comment:
some of these dependencies are defined already above

Reply:
Re-ordered and removed duplicates.